src/uu/pidwait/Cargo.toml

```
@@ -19,6 +19,12 @@ clap = { workspace = true }
 regex = { workspace = true }
 uu_pgrep = { path = "../pgrep" }
 
+[target.'cfg(unix)'.dependencies]
+rustix = { version = "1", default-features = false, features = ["event", "process", "std"] }
+
+[target.'cfg(windows)'.dependencies]
+windows-sys = { version = "0.59", features = ["Win32_Foundation", "Win32_System_Threading"] }
```

I think you can use `workspace = true` as this dependency is defined as a workspace dependency in the root `Cargo.toml`.
```suggestion
windows-sys = { workspace = true, features = ["Win32_Foundation", "Win32_System_Threading"] }
```

---

Thanks for reviewing. It seems that changing `Cargo.toml` in this way does not affect compilation. I am committing it to the PR.